chapters/3-measurement.tex

```diff
@@ -104,6 +111,47 @@ \section{Creating a data map}
 and maintained as the project evolves
 and new data sources, observations, and research questions arise.
 
+We have introduced a framework of classifying data
+as either \textbf{research design variables}\index{research design variables} 
+or \textbf{measurement variables}\index{measurement variables}
+to better explain where in the data map different types of data should be tracked.
+Research design variables are data that maps information about your research subjects to the
+questions that are asked in your research design.
+They can either be variables that often has no meaning outside your research project,
+for example ID variables and treatment status,
+or they can be observations from the real world 
+that determines how each specific research subject 
+should be handled during the analysis, 
+for example treatment uptake and eligibility status.
+Measurement variables are data that
+measures attributes or records responses of research subjects.
+Measurement variables are not controlled by the research team.
+Examples include characteristics of the research subject,
+outcome variables, and control variables among many others.
+
+Research questions can often be expressed in this framework 
+by asking how research design variables correlates with measurement variables.
+This is not identical to dependent and independent variables in a regression.
+For example, research design variables such as treatment status are often not 
+as uncorrelated with real world attributes 
+as the assumptions in our statistical models say that they have to be. 
+One common mitigation of that is to add control variables 
+as additional independent variables in the regressions.
+This helps isolate the relationship between the treatment status and the outcome variable studied. 
+In this example, both the outcome variable (dependent variable) and 
+most control variables (independent variables) 
+are measurement variables that are not direct expressions of research design.
+It is this direct linkage between your theoretical research design 
+to real world characteristics that is the definition of a research variable.
+If you are a PhD economist with several years of experience 
+then this distinction might seem unnecessary to you 
+as this relation is very intuitive for you. 
+However, your research project is likely to depend on contributions 
+from someone to whom this is not as intuitive.
+We believe this distinction is helpful for them, 
+and to maintain this distinction they are handled differently in the data map.
```

Comment:
I don't understand this paragraph well what is being siad in this paragraph. I think it's meant to be a response to David Mckenzie's question about whether by research design and measurement variables we actually mean LHS and RHS variables. I think we can address that by more clearly clarifying in the description (and by renaming as you've done), and then being careful how we use them in explanations throughout. I would suggest to drop this, or at least to significantly revise (i didn't understand well enough to suggest specific revisions).